pmml/pmml.py

```diff
@@ -214,5 +210,5 @@ def print_pmml(pmml_out):
             print("\t\t</Segmentation>", file=pmml_out)
             print("\t</MiningModel>", file=pmml_out)
             print("</PMML>", file=pmml_out)
-except Exception as ioex:
+except IOError as ioex:
```

Comment:
I think the try statement is not needed. If there is an exception, it will throw and print error message anyway. The try statement is used if you have other logic after catch an exception.